text/1055-vanilla-prettier-setup-in-blueprints.md

```diff
@@ -0,0 +1,93 @@
+---
+stage: accepted
+start-date: 2024-12-03T00:00:00.000Z
+release-date: 
+release-versions:
+teams:
+  - cli
+prs:
+  accepted: https://github.com/emberjs/rfcs/pull/1055
+project-link:
+suite: 
+---
+
+# Vanilla Prettier Setup in Blueprints
+
+## Summary
+
+This RFC proposes to migrate to a vanilla Prettier setup in the blueprints, instead of running Prettier via ESLint and Stylelint.
+
+This RFC also proposes to not include any custom values for specific Prettier options, and instead truly rely on Prettier's defaults for formatting.
```

Comment:
We have to have some options configured for gjs/gts, which we support out of the box, as of ember-cli 6.1 (for eslint and prettier)

Reply:
Which options? Do you mean `singleQuote` and `templateSingleQuote`?

Reply:
I think I've covered all of this in the RFC, but let me know if something isn't clear.

Reply:
This has now been dropped from the RFC.